answer/lib.rs

```diff
@@ -50,6 +56,19 @@ impl Type {
         }
     }
 
+    pub fn get_label<'a>(
```

Comment:
Unfortunately it seems like we can't use the inner as an `&dyn TypeAPI`, otherwise this would be extremely easy!
However, most of the time we know the particular type we unwrap to -- this is mostly used in error reporting.